easybuild/easyconfigs/c/CONTRAlign/CONTRAlign-2.01_gcc47.patch

```diff
@@ -0,0 +1,21 @@
+--- contralign.orig/src/Makefile	2008-08-15 03:06:30.000000000 +0200
```

Comment:
@pescobar please add a line of comment explaining the patch, and add yourself as an author?